spec/models/post_action_spec.rb

```diff
@@ -755,11 +755,11 @@ def value_for(user_id, dt)
     end
   end
 
-  it "prevents user to act twice at the same time" do
-    # flags are already being tested
-    all_types_except_flags =
-      PostActionType.types.except(*PostActionType.flag_types_without_custom.keys)
-    all_types_except_flags.values.each do |action|
+  # flags are already being tested
+  all_types_except_flags =
+    PostActionType.types.except(*PostActionType.flag_types_without_custom.keys)
+  all_types_except_flags.values.each do |action|
+    it "prevents user to act twice at the same time" do
```

Comment:
We put the example inside the loop so it resets each time. This is because some flags will hide the post, causing "invalid access" and preventing the user from testing the next flag.